iclr2017/python/get-areachairs.py

```diff
@@ -34,23 +34,26 @@
 
 if args.paper_number!=None:
     
-    try:
+    if notes:
```

Comment:
aren't I just checking to see if the list is empty? if so, this is the "proper" way to do it in python (according to my internet search)

http://stackoverflow.com/questions/53513/best-way-to-check-if-a-list-is-empty